packages/experimental/Button/src/Button.types.ts

```diff
@@ -111,16 +111,62 @@ export interface ButtonProps extends Omit<IWithPressableOptions<ViewProps>, 'onP
   fluid?: boolean;
 }
 
+export interface ButtonMacProps {
+  /*
+   * Name of the icon to show on the Button.
+   */
+  image?: Image;
+  /*
+   * Text to show on the Button.
+   */
+  title?: string;
+  /*
+   * ButtonStyle wrapped enum
+   */
+  buttonStyle?: ButtonMacStyle;
+  /*
+   * A button can be enabled or disabled.
+   */
+  isEnabled?: boolean;
```

Comment:
I also feel like this is somewhat platform specific? Although I don't really see anything in RN that would map to this

Reply:
I don't think this is platform specific, RN also uses isEnabled flag to indicate the button status.

Reply:
Renamed to enabled